packages/components/src/components/page-error/page-error.tsx

```diff
@@ -1,15 +1,30 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isMobile } from '@deriv/shared';
+import DesktopWrapper from '../desktop-wrapper';
+import MobileWrapper from '../mobile-wrapper';
 import ButtonLink from '../button-link/button-link';
-import DesktopWrapper from '../desktop-wrapper/desktop-wrapper.jsx';
-import MobileWrapper from '../mobile-wrapper/mobile-wrapper.jsx';
 import Text from '../text/text';
 
+type TMessageObject = { message: string; has_html?: boolean };
+
+type TPageErrorProps = {
+    buttonOnClick?: () => void;
+    buttonSize?: 'small' | 'medium' | 'large';
+    classNameImage?: string;
+    header: React.ReactNode;
+    image_url?: string;
+    messages: Array<TMessageObject | React.ReactNode>;
+    redirect_labels: string[];
+    redirect_urls?: string[];
```

Comment:
@jim-deriv please also update the props for this component in /deriv-app/packages/components/stories/page-error/README.md 🙂